src/animation/PropertyBinding.js

```diff
@@ -184,9 +191,9 @@ Object.assign( PropertyBinding, {
 
 		return results;
 
-	},
+	}
 
-	findNode: function ( root, nodeName ) {
+	static findNode( root, nodeName ) {
```

Comment:
The below `if` statement contains `"` characters which should not be the case.

Not rebasing makes it more possible produce a regression.

Reply:
@Mugen87 Sorry for the delay- this is now rebased with single quotes.